scripts/deploy-box.proxy.ts

```diff
@@ -1,15 +1,63 @@
 import { ethers, upgrades } from 'hardhat'
+import { Contract } from 'ethers'
+import { writeFileSync } from 'fs'
 
 import config, { NetworkConfig } from '../config'
+import boxContract from '../artifacts/contracts/token/RarePizzasBox.sol/RarePizzasBox.json';
 
 const getChainlinkOracle = (config: NetworkConfig) => {
   const networkName = config.NETWORK.toLowerCase()
   switch (networkName) {
+    case 'mainnet':
+      return config.CHAINLINK_MAINNET_PRICE_FEED
+    case 'goerli':
+      return config.CHAINLINK_GOERLI_PRICE_FEED
     case 'rinkeby':
       return config.CHAINLINK_RINKEBY_PRICE_FEED
+    case 'ropsten':
```

Comment:
Lets add a `default`

Reply:
i want it to blow up if its not provided